Linear_cell_complex/doc/Linear_cell_complex/CGAL/Linear_cell_complex_incremental_builder.h

```diff
@@ -0,0 +1,95 @@
+
+namespace CGAL {
+
+/*!
+\ingroup PkgLinearCellComplexClasses
+
+The auxiliary class `Linear_cell_complex_incremental_builder` supports the incremental
+construction of linear cell complexes.
+
+\tparam LCC a linear cell complex
+*/
+
+template < class LCC >
+class Linear_cell_complex_incremental_builder
+{
+ public:
+  typedef LCC_ LCC;
+  typedef typename LCC::Dart_descriptor             DH;
+  typedef typename LCC::Vertex_attribute_descriptor VAH;
+  typedef typename LCC::Point                       Point_3;
+  typedef typename LCC::size_type                   size_type;
+
+  /// \name Creation
+  /// @{
+
+  /*!
+   * Constructor
+   */
+  Linear_cell_complex_incremental_builder(LCC & alcc);
+
+/// @}
+
+  /*!
+\name Surface Creation
+
+To build a linear cell complex, the following regular expression gives
+the correct and allowed order and nesting of method calls from this
+section:
+
```

Comment:
Maybe add few words about how cells are glued during the construction. For example something like:

When an edge is added in a facet, if the same edge exists in another facet of the same surface, then the two facets are glued along this edge.

When a facet is added, if the same facet exists in another surface, the two surfaces are glued along this facet.